server/autolink/lib_jira_test.go

```diff
@@ -0,0 +1,125 @@
+package autolink_test
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-plugin-autolink/server/autolink"
+)
+
+var jiraTests = []linkTest{
```

Comment:
I don't see the need to put this into a global variable. No other tests needs it.

Reply:
I just want to move the relevant template definitions to the top of the file, as de-facto documentation. Eventually - should be exposed as a bunch of godocs? (Orthogonal to any other efforts to "templatize" autolink deployments)

Reply:
I just like to have everything related to a test bundled into the `TestX` method. But at the end, that is personal preference.

> Eventually - should be exposed as a bunch of godocs?

What is the value in having godocs for tests?